src/MasterPeace/Bundle/BookBundle/DataFixtures/ORM/LoadBookData.php

```diff
@@ -23,6 +24,8 @@ public function load(ObjectManager $manager)
                 ->setCover($bookDetail['cover'])
                 ->setIsbnCode($bookDetail['isbn_code']);
             $manager->persist($book);
+
+            $this->addReference('book' . $id, $book);
```

Comment:
kai pirma kart dariau kazkaip nesigavo. Man rods visa esme buvo tame kad bet koki pakeitima jei padarai fixturose reik dropint ir per nauja updeitint schema. As taip nedarydavau ir man errorai metydavosi. Jei buciau sugalvojes padropint schemas jau pirmadieni vakare buciau imetes visa ta darba ir be tu arrayobjektu :D